css/theme-dark.css

```diff
@@ -1,5 +1,15 @@
 body[theme="dark"] .search-container {
   border-color: #C1C1C1;
+  background-color: var(--dark-background-color);
+}
+
+body[theme="dark"] #search-input,
+body[theme="dark"] .mdc-icon-button {
+  color: white;
+}
+
+body[theme="dark"] #search-counting.nomatches {
+  color: #ff0000;
```

Comment:
why has this changed?

Reply:
I changed that because the red that went with that just didn't look right. Should I change it back?

Reply:
I think we should leave it. I'm wary to make color changes that haven't been approved by a designer/UX review.